src/App/Models/IEzStartup.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace App.Models
```

Comment:
I'd put the interface in a "Common" project. The reason why will become obvious later